formalmethods_intrepid_app_upload_102.py

```
@@ -0,0 +1,36 @@
+from .contexts import contexts
+
+ur = Blueprint('upload', __name__)
+
+@ur.route('', methods=['POST'])
+def upload():
+    """
+    Uploads and parses a file in intrepid syntax
+    """
+    file = request.files['file']
+    filename = secure_filename(file.filename)
+    filepath = '/tmp/' + filename
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)